components/Header/Header.tsx

```diff
@@ -8,39 +8,84 @@ import blockBodyScroll from "utils/block-body-scroll";
 import HeaderCTA from "./HeaderCTA";
 import styles from "./Header.module.css";
 import Magnifier from "./assets/magnify.svg?react";
+import MagnifierMobile from "./assets/magnify-mobile.svg?react";
 import Link from "components/Link";
+import {
+  EventBanner,
+  EventProps,
+  getComingEvent,
+} from "components/EventBanner";
+// @ts-ignore
+import eventData from "../../public/data/events.json";
 
 const Header = () => {
+  const selectedEvent = eventData
+    ? getComingEvent(eventData as EventProps)
+    : null;
   const [isNavigationVisible, setIsNavigationVisible] =
     useState<boolean>(false);
   const toggleNavigaton = useCallback(() => {
     setIsNavigationVisible((value) => !value);
     blockBodyScroll(isNavigationVisible);
   }, [isNavigationVisible]);
-
   return (
-    <header className={styles.wrapper}>
-      <a href="/" className={styles["logo-link"]}>
-        <Logo />
-      </a>
-      <HeadlessButton
-        onClick={toggleNavigaton}
-        className={styles.hamburger}
-        data-testid="hamburger"
-        aria-details="Toggle Main navigation"
-      >
-        <Icon name={isNavigationVisible ? "close" : "hamburger"} size="md" />
-      </HeadlessButton>
-      <div
-        className={cn(styles.content, {
-          [styles.visible]: isNavigationVisible,
-        })}
+    <>
+      {selectedEvent && (
+        <EventBanner initialEvent={selectedEvent as EventProps} />
+      )}
+      <header
+        className={`${styles.wrapper} ${selectedEvent ? styles.margin : " "}`}
       >
-        <Menu />
-        <HeaderCTA />
-      </div>
-    </header>
+        <a href="/" className={styles["logo-link"]}>
+          <Logo />
+        </a>
+        <NavSearch
+          className={styles.navSearchMobile}
+          testid="mobile-search"
+          aria-label="search website"
+        />
```

Comment:
We removed this in https://github.com/gravitational/docs/pull/390/files because having two search components on the docs site was confusing users.

Would you mind explaining why we're restoring it?

Reply:
I wasn't aware that the search link was missing on this repo on purpose and was looking to update the content to to be on par with the other repos. But I'll revert that change in that case.